docs/getting-started.rst

```diff
@@ -130,7 +130,7 @@ class, we're going to add a function and some class attributes.
        def on_update(self, update_event, signal):
            self.position += self.velocity * update_event.time_delta
 
-Now, your sprite should fly off screen.
+Now, your sprite should fall down and off the screen.
```

Comment:
This is different in the version this will release with, so it will fly up in 0.6